net-misc/gofish/gofish-1.2.ebuild

```diff
@@ -19,6 +19,8 @@ RDEPEND=""
 
 S=${WORKDIR}/${PN}
 
+DOCS=( AUTHORS ChangeLog Configure_GoFish README TODO )
```

Comment:
dont define this, just do `dodoc Configure_GoFish` after `default` in `src_install`